datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -165,4 +166,10 @@ SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam
     @ApiOperation(value = "list advertised datalake versions", produces = MediaType.APPLICATION_JSON, nickname = "advertisedruntimes")
     List<AdvertisedRuntime> advertisedRuntimes();
 
+    @POST
+    @Path("/crn/{envCrn}/ranger_cloud_identity_mapping")
```

Comment:
See the path on line 68. it looks like
/envcrn/{envCrn}
This should match to distinguish between the resource crn and the environment crn